config_center/nacos/impl.go

```diff
@@ -65,101 +66,130 @@ func newNacosDynamicConfiguration(url *common.URL) (*nacosDynamicConfiguration,
 }
 
 // AddListener Add listener
-func (n *nacosDynamicConfiguration) AddListener(key string, listener config_center.ConfigurationListener, opions ...config_center.Option) {
-	n.addListener(key, listener)
+func (nacos *nacosDynamicConfiguration) AddListener(key string, listener config_center.ConfigurationListener, opions ...config_center.Option) {
```

Comment:
pls do not change the receiver name. because it is used in Go way which should be a as simple as possible.

in my opinion, if u think there are so many different nacosDynamicConfiguration receiver name, what u should do is change them in a unified name like "n" or "c".